frontend/src/features/leases/detail/LeaseContainer.tsx

```diff
@@ -83,7 +84,7 @@ export const LeaseContainer: React.FunctionComponent<ILeaseAndLicenseContainerPr
     <>
       <LeaseLayout>
         <LeaseBreadCrumb
-          leaseId={props.match.leaseId}
+          leaseId={lease?.id}
```

Comment:
small improvement - we already have a **lease** var here. No need to go back to the URL params